remote/startup.sh

```diff
@@ -16,6 +16,7 @@ else
 fi
 
 python3 /opt/setup_workspace.py;
+stat /opt/setup_workspace_purevariants.py &>/dev/null && python3 /opt/setup_workspace_purevariants.py
```

Comment:
Please add this in the `Dockerfile` of `pure-variants` (e.g. add it in line 5 of the startup.sh).

Reply:
I think that using `stat` to check if the file exists before executing it is better (more easily debbugable) than injecting the instruction from the Dockerfile with a `sed` command. However, I can switch to this approach if you prefer.